public/app/core/utils/ticks.ts

```diff
@@ -25,3 +25,43 @@ export function tickStep(start: number, stop: number, count: number): number {
 
   return stop < start ? -step1 : step1;
 }
+
+export function getScaledDecimals(decimals, tick_size) {
```

Comment:
Yes, I want to recreate flot.js behaviour while working with axis ticks. So I have no another chance to do that. This code calculates scaledDecimals for the value formatter.